.circleci/config.yml

```diff
@@ -1,21 +1,63 @@
 version: 2.1
 
 commands:
+  create_concatenated_package_lock:
+    description: "Concatenate all package-lock.json files recognized by lerna.js into single file. File is used as checksum source for part of caching key."
+    parameters:
+      filename:
+        type: string
+    steps:
+      - run:
+          name: Combine package-lock.json files to single file
+          command: |
+            npx lerna list -p -a | awk -F packages '$2!="" {printf "\"packages%s/package-lock.json\" ", $2}' | xargs cat > << parameters.filename >>
+
   install-deps:
+    parameters:
+      onlyInstall:
+        type: boolean
+        default: false
+
     steps:
       - checkout
       - run: curl -sL https://sentry.io/get-cli/ | bash
       - restore_cache:
           keys:
-            - v1-deps-{{ checksum "package-lock.json" }}
-            - v1-deps
-      - run: |
-          npm -v
-          npm install
+            - v2-deps-{{ checksum "package-lock.json" }}
+            - v2-deps
```

Comment:
The caching strategy actually changed significantly since now we added a new prefix `sub-deps`. I tested the CI, and it seems like if the old v1 prefix is still been used, unexpected cache retrieval would occur. The documentation recommends using a new prefix.